src/features/pageObjects/utils/pageObjectTemplate.js

```diff
@@ -1,20 +1,11 @@
-import CyrillicToTranslit from "cyrillic-to-translit-js";
```

Comment:
please, remove it from package.json too

Reply:
> please, remove it from package.json too

it's already here [https://github.com/jdi-testing/jdn-ai/pull/964/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L51](package.json 51 line)